setup.py

```diff
@@ -1,10 +1,10 @@
 from setuptools import setup
 
-requirements = ["dataclasses==0.8;python_version<'3.7'"]
+requirements = ["dataclasses==0.8,~=0.6;python_version<'3.7'"]
 
 setup(
     name="clr",
-    version="0.3.0",
+    version="0.3.2",
```

Comment:
What happened to `0.3.1`?

Reply:
Was tagged as a github release but not updated here